src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id)
+                .orElseThrow(() ->
+                        new RuntimeException("Couldn't get an actor from DB with id " + id));
```

Comment:
```suggestion
                        new RuntimeException("Couldn't get an actor from DB with id " + id));
```
Better use more specific exceptions for example EntityNotFoundException